README.md

```diff
@@ -24,6 +24,130 @@ Then run it with:
 cargo supply-chain publishers
 ```
 
+
+<p><br>
+# Command summary<br>
```

Comment:
This doesn't seem to render as a header on Github.

Reply:
Yea, I'll rework that bit. We'll see.